lib/application/index.js

```diff
@@ -4,6 +4,9 @@
  * configuration and rendering.
  */
 
+// Local
+var Promise = require('../polyfills/promise');
```

Comment:
Yea. The real question here is whether we want bv-ui-core to automatically include the polyfill.